src/drivers/boards/px4fmu-v4/px4fmu_spi.c

```diff
@@ -103,7 +103,16 @@ __EXPORT void stm32_spi1select(FAR struct spi_dev_s *dev, enum spi_dev_e devid,
 		px4_arch_gpiowrite(GPIO_SPI_CS_MPU9250, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_HMC5983, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_MS5611, 1);
-		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_20608_G, !selected);
+		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_2060X_G, !selected);
+		break;
+
+	case PX4_SPIDEV_ICM_20602:
```

Comment:
If both IC are out in "the wild". Then I make sense to support both chips on this board and the AUAV2.1 Try to start the new one first if it fails try to start the old one. In both cases (no pun intended) it is one case statement with 2 labels. There is one GPIO defined.